### PR TITLE
Docs/command line arguments doc 361

### PR DESCRIPTION
**Pull request by eddyrainy**
DOC ticket: https://improbableio.atlassian.net/browse/DOC-361
Pending: *protocol logging* (https://github.com/improbable/everything/pull/3814/files) needs to be published first.

The local build layout:
image
image
image

### CODE REVIEW

**Comment by ernopp** [comment]
great job!!

File: SpatialGDK/Documentation/content/command-line-arguments.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        <tr>
+            <td style="width: 30%;"><strong>Option</strong></td>
+            <td style="width: 8%;"><strong>Type</strong></td>
+            <td style="width: 62%;"><strong>Description</strong></td>
+        </tr>
+        <tr>
+            <td style="width: 30%;"><code>locator<code></td>
+            <td style="width: 8%;">flag</td>
+            <td style="width: 62%;">
+                <p dir="ltr">To connect a client-worker instance to a deployment using Locator, add this flag to the <code>ClientTravel</code> URL.</p>Note: If you add this option, ensure that you also add the <code>playeridentity</code> option and the <code>login</code> option to the URL.<br></td>
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/command-line-arguments.md]:
```suggestion
                <p dir="ltr">To connect a client-worker instance to a deployment using the Locator, add this flag to the <code>ClientTravel</code> URL.</p><strong>Note</strong>: If you add this option, ensure that you also add the <code>playeridentity</code> option and the <code>login</code> option to the URL.<br></td>
```

File: SpatialGDK/Documentation/content/command-line-arguments.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+            <td style="width: 8%;">string</td>
+            <td style="width: 62%;">The <a href="https://docs.improbable.io/reference/13.7/shared/auth/integrate-authentication-platform-sdk#2-generating-a-playeridentitytoken"><code>PlayerIdentityToken</code></a> (PIT) that the client-worker instance passes to the Locator service to prove its player identity within SpatialOS.<br></td>
+        </tr>
+        <tr>
+            <td style="width: 30%;"><code>login<code><br></td>
+            <td style="width: 8%;">string</td>
+            <td style="width: 62%;">The <a href="https://docs.improbable.io/reference/13.7/shared/auth/integrate-authentication-platform-sdk#3-generating-a-logintoken"><code>LoginToken</code></a> that the client-worker instance passes to the Locator service to be permitted to join the deployment.<br></td>
+        </tr>
+        <tr>
+            <td style="width: 30%;"><code>useExternalIpForBridge</code><br></td>
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/command-line-arguments.md]: It looks like this entry is the same as the one in "Connection arguments" - is this intentional?

Comment (eddyrainy) [reply SpatialGDK/Documentation/content/command-line-arguments.md]: The option belongs specifically to *Map travel URL options*, not to *Connection arguments*. The usage of it is different as well (without `-` in the option Vs with `-` in the argument in the *Connection arguments*).

Comment (anne-edwards) [reply SpatialGDK/Documentation/content/command-line-arguments.md]: Ah OK, thanks for the explanation! Makes sense :)

File: SpatialGDK/Documentation/content/map-travel.md [hunk 3]
```diff
@@ -127,3 +127,9 @@ The `LaunchClient.bat` (which we have provided) already includes the local host
 When launching a client-worker from the SpatialOS Console using the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher), the client-worker will connect to SpatialOS by default. It has the `Locator` information required to connect to said deployment included as command-line arguments. When these `Locator` arguments are present, client-workers will attempt to connect automatically. Please note the launcher login tokens are only valid for 15 minutes.  
 
 > Connecting by default when using the launcher is subject to change.
+
+<br/>
+
+<br/>------------<br/>
+2019-06-13 Page updated with editorial review: Added Locator information
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/map-travel.md]:
```suggestion
2019-06-13 Page updated with limited editorial review: Added Locator information
```

File: SpatialGDK/Documentation/toc.md [hunk 4]
```diff
@@ -37,6 +37,7 @@
         - [Non-Unreal server-worker types]({{urlRoot}}/content/non-unreal-server-worker-types)
     - [Snapshots]({{urlRoot}}/content/spatialos-concepts/generating-a-snapshot) 
     - [Gameplay Ability System]({{urlRoot}}/content/ability-system)
+    - [Command line arguments]({{urlRoot}}/content/command-line-arguments)
```
Comment (ElleEss) [inline SpatialGDK/Documentation/toc.md]: I think this page should go with Helper scripts, Directory structure as it's about Workflows and the mechanics of using the GDK.

**Pull request merged**